### PR TITLE
data:Megagig

### PR DESCRIPTION
**Pull request by Megagig**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/Megagig.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Obi Anthony",
+  "displayStatsPublic": true,
```
Comment (sital002) [inline data/Megagig.json:3]:
```suggestion
```
This property isnot required anymore

File: data/Megagig.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "type":"personal",
```
Comment (sital002) [inline data/Megagig.json:4]:
```suggestion
```
type property isnot required

File: data/Megagig.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  "bio":"I am a Full Stack Developer",
+  "avatar": "https://github.com/Megagig.png",
```
Comment (sital002) [inline data/Megagig.json:6]:
```suggestion
```
avatar property isnot required

**Review by sital002 — DISMISSED** [review]
Please make the necessary changes to get your profile merged

**Review by SaraJaoude — APPROVED** [review]
I have committed the changes to make the Json pass and will now merge this PR.

**Pull request merged**